Services/DataBaseService/DbModels/DbUserCredential.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace DataBaseService.DbModels
+{
+    public class DbUserCredential
+    {
+        public Guid Id { get; set; }
+        public string Email { get; set; }
+        public string PasswordHash { get; set; }
+    }
+
+    public class UserConfiguration : IEntityTypeConfiguration<DbUserCredential>
```

Comment:
Is it ok to have these 2 classes in the same file?

Reply:
This has already been discussed with Dasha. Spartak gave advice to put this stuff in one file